Packs/Malware/Scripts/IsolationAssetWrapper/IsolationAssetWrapper.yml

```diff
@@ -0,0 +1,86 @@
+args:
+- default: false
+  description: 'Device ids to isolate or unisolate'
+  isArray: true
+  name: device_ids
+  required: true
+  secret: false
+- default: false
+  description: 'The action to do with device ids - isolate or unisolate'
+  isArray: false
+  name: action
+  required: true
+  secret: false
+  auto: PREDEFINED
+  predefined:
+  - 'isolate'
+  - 'unisolate'
+comment: 'This is a wrapper to isolate or unisolate hash list from XDR, MSDE or CrowdStrike (Available from Cortex XSOAR 6.0.0).'
+commonfields:
+  id: IsolationAssetWrapper
+  version: -1
+enabled: false
+name: IsolationAssetWrapper
+outputs:
+  # MSDE outputs
+- contextPath: MicrosoftATP.MachineAction.ID
+  description: The machine action ID.
+  type: String
+- contextPath: MicrosoftATP.MachineAction.Type
+  description: Type of the machine action.
+  type: String
+- contextPath: MicrosoftATP.MachineAction.Scope
+  description: Scope of the action.
+  type: Unknown
+- contextPath: MicrosoftATP.MachineAction.Requestor
+  description: The ID of the user that executed the action.
+  type: String
+- contextPath: MicrosoftATP.MachineAction.RequestorComment
+  description: Comment that was written when issuing the action.
+  type: String
+- contextPath: MicrosoftATP.MachineAction.Status
+  description: The current status of the command.
+  type: String
+- contextPath: MicrosoftATP.MachineAction.MachineID
+  description: The machine ID on which the action was executed.
+  type: String
+- contextPath: MicrosoftATP.MachineAction.ComputerDNSName
+  description: The machine DNS name on which the action was executed.
+  type: String
+- contextPath: MicrosoftATP.MachineAction.CreationDateTimeUtc
+  description: The date and time when the action was created.
+  type: Date
+- contextPath: MicrosoftATP.MachineAction.LastUpdateTimeUtc
+  description: The last date and time when the action status was updated.
+  type: Date
+- contextPath: MicrosoftATP.MachineAction.RelatedFileInfo.FileIdentifier
+  description: The file identifier.
+  type: String
+- contextPath: MicrosoftATP.MachineAction.RelatedFileInfo.FileIdentifierType
+  description: 'The type of the file identifier. Possible values: "SHA1" ,"SHA256", and "MD5".'
+  type: String
+
+
+    # XDR outputs
+
+- contextPath: PaloAltoNetworksXDR.Isolation.endpoint_id
+  description: The endpoint ID.
+  type: String
+- contextPath: PaloAltoNetworksXDR.UnIsolation.endpoint_id
+  description: Isolates the specified endpoint.
+  type: String
+
+
+    # CrowdStrike outputs
```

Comment:
?

Reply:
I added a comment for each brand.
In this command there are currently no outputs related to crowdstrike falcon, so it's empty.